### DISCUSSION TITLE
When can we expect Ogmios v6 (stable) to be released?

### DISCUSSION THREAD

**Discussion by gytis-ivaskevicius** [CardanoSolutions/ogmios#352] [Q & A] [answered]
It would be nice to have a tracking issue on this, also it would be nice if we had release candidate OCI images with `/config` directory since it is a little bothersome to work with it as of right now

**Comment by KtorZ** [comment]
As soon as we have a stable Conway era in the node. Note that Ogmios v6 can be treated as stable for any eras other than Conway. So if you aren't integrating with SanchoNet at the moment, there's nothing more to expect.

**Reply by gytis-ivaskevicius** [reply]
Yes, we are integrating with Sanchonet, tracking issue would be nice but it does not sound like its going to happen. Could we at least have ogmios images with prepopulated `/config` for release candidates by any chance?

**Reply by KtorZ** [reply, marked as answer]
This should be the case already, though the config path has slightly changed compared to v5. The image now embeds configs for all networks. 

Please refer to the user manual (https://ogmios.dev/getting-started/docker/#configuration) for detail. 

(Edit: it isn't actually mentioned in the doc, but this includes config files for SanchoNet)
